src/Meshes/abstractMesh.ts

```diff
@@ -18,6 +18,7 @@ import { MaterialDefines } from "../Materials/materialDefines";
 import { Light } from "../Lights/light";
 import { Skeleton } from "../Bones/skeleton";
 import { MorphTargetManager } from "../Morph/morphTargetManager";
+import { BakedVertexAnimationManager } from "../BakedVertexAnimation/bakedVertexAnimationManager";
```

Comment:
We should not import but declare it
declare type BakedVertexAnimationManager = import("../BakedVertexAnimation/bakedVertexAnimationManager").BakedVertexAnimationManager ;


cc @RaananW to make sure it will not be a problem for tree shaking and side effects

Reply:
I would recommend to expose an interface actually and use that interface here

Reply:
Morphtarget is part of what is problematic for treeshaking and side effects (same as skeleton)

Reply:
Added an interface. I was not sure what should be there, take a look at let me know if something is missing/too much.

Reply:
If it is only used as an interface typescript will remove it from the final .js file, so it should be fine.

Reply:
yup but the declare prevents any maintenance misuse of the imported file :-) The interface is great I agree